spec/models/post_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) {User.create!(first_name:'s',last_name:'j', email: 'jay2@he.com',password:'foobar',password_confirmation:'foobar')}
+
+  it "expects post to be valid" do
+    post = Post.create!(content: "the test post", author_id: user.id).save
+    expect(post).to eq(true)
+  end
+
+  it 'expects author to be invalid' do
+    post = Post.new(content: "the test post").save
+    expect(post).to eq(false)
+  end
+
+  it 'expects content to be invalid' do
+    post = Post.new(author_id:user.id).save
+    expect(post).not_to eq(true)
+  end
+
+  it 'expects post to be stored' do 
```

Comment:
We don't need to test whether or not a post will be saved to the database. If the validations pass, then they'll surely be saved. The validation tests are enough for this